README.rst

```diff
@@ -113,13 +113,20 @@ OSX
 Ubuntu/Debian 16.10+
 ^^^^^^^^^^^^^^^^^^^^
 
-Ubuntu starting at 16.10 supports Python 3.6 in the official
-repositories, and you can just install Python 3.6 and all the system
-dependencies like this:
+Ubuntu starting at 16.10 supports Python 3.6+ in the official repositories. We recommend
+using Python 3.7. 
```

Comment:
I'm not sure we want to recommend Python 3.7. Perhaps just remove that sentence.

Reply:
I added this because the rest of my instructions were for Python 3.7. The whole sentence could be removed.